support/Pipelines/Bbh/Inspiral.py

```diff
@@ -64,12 +64,12 @@ def _constraint_damping_params(
 ) -> dict:
     total_mass = mass_left + mass_right
     return {
-        "Gamma0Constant": 0.001 / total_mass,
+        "Gamma0Constant": 0.01 / total_mass,
         "Gamma0LeftAmplitude": 4.0 / mass_left,
         "Gamma0LeftWidth": 7.0 * mass_left,
         "Gamma0RightAmplitude": 4.0 / mass_right,
         "Gamma0RightWidth": 7.0 * mass_right,
-        "Gamma0OriginAmplitude": 0.075 / total_mass,
```

Comment:
I'm not sure why this differs from SpEC, I believe it should be 0.075, and I pushed a fixup for this maybe a year ago now. I changed it back here since this is what was done in the paper, I'm not sure why it was done or if it changes the result in a significant way. @geoffrey4444 do you know if this impacts the bbh runs?

Reply:
Yes I remember this being changed from 0.75 down to 0.075, but I believe this was also when we were running with too low of resolution and we weren't convergent. Once our resolution was convergent, this new value seemed to decrease the spikiness of the constraint violations . But @geoffrey4444 please confirm

Reply:
I used 0.75 instead of 0.075 as in SpEC, in the bbh inspirals for the paper, because I found that this helped with reducing the big burst of constraints from junk radiation. I also set the constant to 0.01 instead of 0.001 as in SpEC, e.g. https://arxiv.org/src/2410.00265v1/anc/Bbh_Lev1_Inspiral.yaml

I'm not saying those are the perfect numbers, but they worked well for the runs in the paper. I just multiplied these two numbers by 10 and the constraints decayed better after the initial junk burst. In ParaView I saw large constraints far from the inner shells, around the envelope part of the domain, so I focused on these terms, rather than the terms controlling constraints close to the excision surfaces